src/App.tsx

```diff
@@ -1,25 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import './styles/App.css';
+import './styles/nav-top.css';
+import './styles/nav-middle.css';
+import './styles/nav-bottom.css';
+import './styles/bottom-bar.css';
+import './styles/filter.css';
+import Header from "./components/Header";
+import Nav from "./components/Nav";
+import Footer from "./components/Footer";
+import ShopPage from "./pages/shop/ShopPage";
+import CartPage from "./pages/cart/CartPage";
+import SingleBeerPage from "./pages/singlebeer/SingleBeerPage";
+import HomePage from './pages/home/HomePage';
+import ConnectionErrorPage from './pages/conectionError/ConnectionErrorPage';
+import ErrorPage from './pages/error/ErrorPage';
+import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
+import getNumberOfProuductsInCart from './helpers/getNumberOfProuductsInCart';
+
 
 function App() {
+  
+  const [numberOfProductsInCart, setNumberOfProductsInCart] = useState(getNumberOfProuductsInCart())
```

Comment:
this single data is controlled from both local storage and the other components. Data flow should be one directional

Reply:
Fixed, now its getting controlled only by local storage.